examples/swarm-setup.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	metrics "github.com/ipfs/go-libp2p/p2p/metrics"
+	tu "github.com/ipfs/go-libp2p/testutil"
+	ma "github.com/jbenet/go-multiaddr"
+	swarm "go-libp2p/p2p/net/swarm"
```

Comment:
you'll also need to change the multiaddr import to use the vendored one:
```
	ma "gx/QmVUi2ncqnU48zsPgR1rQosDGwY3SSZ1Ndp33j33YjXdsj/go-multiaddr"
```

This is because of go's stupid insistence that the import path of a type is part of the type.